example/android/app/build.gradle

```diff
@@ -39,8 +39,8 @@ android {
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
         applicationId "com.example.example"
-        minSdkVersion 16
-        targetSdkVersion 28
+        minSdkVersion 21
```

Comment:
While I am a big fan of moving apps to have Android 5 as min supported version, but I believe not all people will like it, so I would suggest to either revert this line change or make value `19`, so it matches the official support level: https://docs.flutter.dev/development/tools/sdk/release-notes/supported-platforms#supported-platforms

Reply:
thx! will do that